ibis/backends/tests/test_numeric.py

```diff
@@ -507,7 +510,15 @@ def test_random(con):
     ],
 )
 @pytest.mark.notimpl(
-    ["sqlite", "clickhouse", "datafusion", "impala", "mysql", "postgres"]
+    [
+        "clickhouse",
+        "duckdb",
+        "datafusion",
+        "impala",
+        "mysql",
+        "postgres",
+        "sqlite",
```

Comment:
I'm not sure why all these are missing a clip implementation, but separately we should implement that.